src/lib/renderComponent.tsx

```diff
@@ -34,27 +33,18 @@ export interface IRenderResultConfig<P> {
 
 export type RenderComponentCallback<P> = (config: IRenderResultConfig<P>) => any
 
-type IRenderConfigProps = {
-  [key: string]: any
-  variables?: ComponentVariablesInput
-  styles?: IComponentPartStylesInput
-}
-
 export interface IRenderConfig {
   className?: string
   defaultProps?: { [key: string]: any }
   displayName: string
   handledProps: string[]
-  props: IRenderConfigProps
-  state: { [key: string]: any }
+  props: IProps
+  state: IState
 }
 
-const getAccessibility = <P extends {}>(props, state) => {
+const getAccessibility = props => {
   const { accessibility: customAccessibility, defaultAccessibility } = props
-  return callable(customAccessibility || defaultAccessibility || DefaultBehavior)({
-    ...props,
-    ...state,
-  })
+  return callable(customAccessibility || defaultAccessibility || DefaultBehavior)(props)
```

Comment:
accessibility functions end up with being provided with state only now - but previously `state` data was provided as well (this was a correct and expected behavior)

Reply:
@jurokapsiar is this correct here?
@kuzhelov anything to change?

Reply:
accessibility need a merge of props and state. props should override state. looking at line 83, I think this is correctly implemented now.

Reply:
I would rather change the name of the argument, because it is a bit misleading - to indicate that 'state' is merged to 'props'

Reply:
done, thanks guys